src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -260,7 +260,7 @@ export class ComponentTestUtils<T extends MyGameComponent> {
     }
     public async expectInterfaceClickSuccess(elementName: string): Promise<void> {
         const element: DebugElement = this.findElement(elementName);
-        expect(element).withContext('Element "' + elementName + '" should exist.').toBeTruthy();
```

Comment:
point enlevé parce que sinon ce qui s'affiche c'est « Error: Element blibli should exist.: expected null to be truthy"